packages/react-query/src/__tests__/suspense.test.tsx

```diff
@@ -332,9 +335,20 @@ describe("useQuery's in Suspense mode", () => {
     fireEvent.click(rendered.getByText('retry'))
 
     await waitFor(() => rendered.getByText('rendered'))
+
+    // check if the console.error function was called
+    if (consoleMock.mock.calls[0]) {
+      //test the error message that was passed to the console
+      expect(consoleMock.mock.calls[0][0]).toContain('Suspense Error Bingo')
+    }
```

Comment:
why a conditional expect? If the console mock is never called, we just don't expect anything and the test will run through fine ...

Reply:
There were stack traces being logged as well, so the mock console was being called many times because of that, i was checking only the first error with this.

Reply:
even then, the `if` statement is unnecessary.

I've tried and this seems to work well:

```
expect(consoleMock).toHaveBeenCalledWith(
  expect.objectContaining(new Error('Suspense Error Bingo')),
)
```